runelite-api/src/main/java/net/runelite/api/VarClientInt.java

```diff
@@ -36,6 +36,8 @@
 {
 	TOOLTIP_TIMEOUT(1),
 
+	MEMBERSHIP_STATUS(103),
```

Comment:
Shouldn't this be memebership days? Or well, shouldn't we use that? Friend list plugin is using it

Reply:
Membership days doesn't get set unless you hit the "Welcome to RuneScape" screen

Reply:
Why are we not using that in the friend list then?

Reply:
We probably should be, this one is more reliable haha